Signal/src/util/OWSBackupExportJob.m

```diff
@@ -322,15 +363,38 @@ - (BOOL)exportDatabase
                                              copiedInteractions++;
                                              copiedEntities++;
                                          }];
+
+            // Copy migrations.
+            [srcTransaction
```

Comment:
Per our offline convo, this might change.  Much will depend on if like Android we _don't_ use YapDatabase/SQLCipher but another representation for our database snapshots, which would preclude your clever idea of running the migrations on the snapshot database.